charts/nautobot/values.yaml

```diff
@@ -516,6 +516,38 @@ nautobots:
 #     - "/graphql"
 #   initNautobot: false
 
+# Default values for Network Policy
+# @default -- See values.yaml
+networkPolicy:
+  # -- [[ref](https://kubernetes.io/docs/concepts/services-networking/network-policies/)]Enables a Network Policy for Nautobot pods
+  enabled: false
+  # -- Permit all Nautobot pods to access any destination in any port
+  allowAllEgress: false
+  # -- Allow outside connectivity to http/https ports of Nautobot Web
+  allowExternalIngress: true
+  # -- Add extra Egress rules to the Network Policy
+  extraEgress: []
+  # e.g.
+  # extraEgress:
+  #   - to:
+  #       - namespaceSelector: {}
+  #         podSelector:
+  #           matchLabels:
+  #             k8s-app: kube-dns
+  #     ports:
+  #       - port: 53
+  #         protocol: UDP
```

Comment:
Let's say I want to just add another port. How would you do it?

```
extraEgress:
  - ports:
       - port: 8888
          protocol: "TCP"
```
?

Reply:
Yes as you already mentioned: Similar example: 
```
extraEgress:
  - ports:
    - port: 80
      protocol: TCP
```